frontend/src/devicelist/ActionMenu.jsx

```diff
@@ -1,8 +1,10 @@
 import React from "react";
+import { NavLink } from "react-router-dom";
```

Comment:
Not used. Remove

Reply:
Not related to my PR, so it would be better to address this issue in @gdzooks's #248 PR, since these changes were added to my PR because I branched off their feature to work on my PR.